scripts/download_data.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-# After running this script all data is in datasets
+# After running this script all data is in datasets. Assumes script is run from base directory
```

Comment:
+1 for @lilleswing's suggestion. @LRParser, could you make the change? Should be good to merge after.